Realm/RLMUtil.hpp

```diff
@@ -139,6 +144,16 @@ static inline realm::BinaryData RLMBinaryDataForNSData(__unsafe_unretained NSDat
     return realm::BinaryData(bytes, data.length);
 }
 
+// Date convertion utilities
+static inline NSDate *RLMDateTimeToNSDate(realm::DateTime dateTime) {
+    return [NSDate dateWithTimeIntervalSince1970:dateTime.get_datetime()];
+}
+
+static inline realm::DateTime RLMDateTimeForNSDate(__unsafe_unretained NSDate *const date) {
+    int64_t time = date.timeIntervalSince1970;
+    return realm::DateTime(time);
```

Comment:
The conversion from `NSTimeInterval` to `int64_t` is still implicit. A cast would make it explicit.